diff -rup lohit-oriya-2.4.3/Lohit-Oriya.sfd lohit-oriya-2.4.3_mod/Lohit-Oriya.sfd
--- lohit-oriya-2.4.3/Lohit-Oriya.sfd	2011-02-08 15:33:15.911147078 +0530
+++ lohit-oriya-2.4.3_mod/Lohit-Oriya.sfd	2011-02-08 15:32:56.659927767 +0530
@@ -13,13 +13,14 @@ Descent: 246
 LayerCount: 2
 Layer: 0 1 "Back"  1
 Layer: 1 1 "Fore"  0
+NeedsXUIDChange: 1
 XUID: [1021 808 1214965479 15809157]
 FSType: 0
 OS2Version: 1
 OS2_WeightWidthSlopeOnly: 0
 OS2_UseTypoMetrics: 1
 CreationTime: 1158849336
-ModificationTime: 1282212414
+ModificationTime: 1297159338
 PfmFamily: 17
 TTFWeight: 400
 TTFWidth: 5
@@ -967,8 +968,8 @@ NameList: Adobe Glyph List
 DisplaySize: -48
 AntiAlias: 1
 FitToEm: 1
-WinInfo: 2884 28 11
-BeginChars: 65864 424
+WinInfo: 2744 28 11
+BeginChars: 65864 425
 
 StartChar: .notdef
 Encoding: 65536 -1 0
@@ -26634,5 +26635,38 @@ SplineSet
 EndSplineSet
 Validated: 33
 EndChar
+
+StartChar: uni20B9
+Encoding: 8377 8377 424
+Width: 629
+VWidth: 0
+Flags: HW
+LayerCount: 2
+Fore
+SplineSet
+358 499 m 0,0,1
+ 359 508 359 508 359 513 c 1,2,-1
+ 59 513 l 1,3,-1
+ 85 576 l 1,4,-1
+ 346 576 l 1,5,6
+ 340 588 340 588 331 598 c 0,7,8
+ 280 653 280 653 165 653 c 2,9,-1
+ 59 653 l 1,10,-1
+ 85 716 l 1,11,-1
+ 583 716 l 1,12,-1
+ 557 653 l 1,13,-1
+ 367 653 l 1,14,15
+ 406 630 406 630 420 576 c 1,16,-1
+ 583 576 l 1,17,-1
+ 557 513 l 1,18,-1
+ 429 513 l 1,19,20
+ 429 357.613 429 357.613 171 339 c 1,21,-1
+ 507 -1 l 1,22,-1
+ 407 -1 l 1,23,-1
+ 70 336 l 1,24,-1
+ 70 395 l 1,25,26
+ 345.891 393 345.891 393 358 499 c 0,0,1
+EndSplineSet
+EndChar
 EndChars
 EndSplineFont
