diff -rup lohit-oriya-2.4.3/Lohit-Oriya.sfd lohit-oriya-2.4.3_mod/Lohit-Oriya.sfd
--- lohit-oriya-2.4.3/Lohit-Oriya.sfd	2010-10-25 11:59:41.000000000 +0530
+++ lohit-oriya-2.4.3_mod/Lohit-Oriya.sfd	2010-10-25 11:57:22.000000000 +0530
@@ -7497,6 +7497,7 @@ SplineSet
  650 476 l 1,124,125
 EndSplineSet
 Validated: 517
+Ligature2: "'akhn' Akhand in Oriya lookup 1 subtable" U0B19 U0B15 U0B4D U0B37 U0B4D
 Ligature2: "'pres' Pre Base Substitutions in Oriya lookup 85-2" U0B19_U0B4D_U0B15.blwf.abvs U0B4D U0B37
 Ligature2: "'pres' Pre Base Substitutions in Oriya lookup 85-1" U0B19_U0B4D_U0B15.blwf.abvs U0B37 U0B4D
 LCarets2: 2 0 0 
